doc/source/notebooks/tailor/external-mean-function.pct.py

```diff
@@ -39,14 +39,23 @@
 # for reproducibility of this notebook:
 np.random.seed(1)
 tf.random.set_seed(2)
+
 # %matplotlib inline
 
 # %% [markdown]
 # ## Generate the tasks
 # To generate the meta and test tasks, we sample from a Gaussian process with an Squared Exponential covariance function and a sinusoidal mean function. Each task is a realization of this process.
 
+
 # %%
-def generate_data(num_functions=10, N=1000):
+def generate_data(num_functions=10, N=50):
```

Comment:
I thought you suggested to go with N=500 ?
would it be worth simply having a cell towards the top of the notebook that sets the three :
num_meta, num_test, N ?
Maybe we can give them more explicit names? num_meta_tasks, num_test_tasks, num_data_per_task (please correct me if I misinterpreted any!)?

Reply:
Thanks for taking a look! I've changed the name, there is a cell in the notebook that sets the three with those names now